repo_e/unique_value.py

```diff
@@ -0,0 +1,5 @@
+def unique(numbers):
+    return set(numbers)
```

Comment:
works well but please return it as a list (right now you accept list and return set - you should accept list and return list)